frontend/src/components/explorer/gemBrowser/Compartment.vue

```diff
@@ -114,6 +114,19 @@ export default {
       return l.join('');
     },
   },
+  metaInfo() {
+    if (!this.model || !this.compartment.name) {
+      return {};
+    }
+
+    const title = `${this.compartment.name}, Compartment in ${this.model.short_name}`;
```

Comment:
Although I can find `title` in the Inspector page, it seems it can not be recognized by search engine nor Slack, with this url: https://dev-metatlas.csbi.chalmers.se/explore/Human-GEM/gem-browser/gene/ENSG00000136010

Reply:
Thanks! See https://github.com/MetabolicAtlas/MetabolicAtlas/issues/614#issuecomment-862290851